### ISSUE TITLE
Filter does not recognise Boolean field 

### ISSUE THREAD

**Issue by Nerian** [code-and-effect/effective_datatables#111]
Hi,

I found a bug: Boolean fields do not work in the filters.
```
filters do
    filter :example, true
end
```
That gives me the following error:

````
undefined method `boolean_field' for #<Effective::FormBuilder:0x00007f917d9b1958>
````
Maybe it should use `check_box`, or a select with True/False – I tend to prefer this one.

**Comment by matt-riemer** [comment]
This has been fixed in effective_datatables 4.5.4

https://github.com/code-and-effect/effective_datatables/commit/d24162d0e0cdff859f3bcc12269263a2bbcf2507

Thanks as always :)

**Comment by Nerian** [comment]
You are da best!
